src/STRPAK.CPP

```diff
@@ -1332,6 +1332,22 @@ BOOL strMatch(					// string match
 			bSeenDifSpace = TRUE;		// both space but different
 	}
 }			// strMatch
+//-----------------------------------------------------------------------------
+#ifndef _WIN32
+int memicmp(			// Compares memory ingnoring case sensitivity
```

Comment:
Where is this used in the code? It might help to add a comment that explains and why this is necessary as opposed to the other string compare functions?

Reply:
`memicmp` is used in 7 different places in CSE. I am adding this function because the first encounter is [here](https://github.com/cse-sim/cse/blob/master/src/ANCREC.CPP#L1258). I added a comment detailing the difference between `stricmp` vs `memicmp`. There is a great explanation in [this post].(https://stackoverflow.com/questions/13095513/what-is-the-difference-between-memcmp-strcmp-and-strncmp-in-c)

Reply:
I just want to confirm that we need `memicmp`, or if we can convert them all to `stricmp`. That is, are there places where the count is not also the length of the string?

Reply:
@galanCA did you find an answer to this? Also, how is it different from `_strnicmp`? You added in #257?

Reply:
That would make sense to me. @chipbarnaby any heartburn?